deps/libgit2.mk

```diff
@@ -77,10 +77,10 @@ $(LIBGIT2_SRC_PATH)/libgit2-bundled_zlib.patch-applied: $(LIBGIT2_SRC_PATH)/sour
 	echo 1 > $@
 
 $(build_datarootdir)/julia/cert.pem:
-	$(JLDOWNLOAD) $(shell pwd)/cacert-2018-01-17.pem https://curl.haxx.se/ca/cacert-2018-01-17.pem
```

Comment:
Might be good to extract this version into it's own variable that can be set in `Make.user`